### DISCUSSION TITLE
Selecting relevant passages on a mobile browser

### DISCUSSION THREAD

**Discussion by waffelbaellchen** [sebastian-hofstaetter/teaching#54] [Advanced IR: Exercise Q&A] [answered]
Hi, I am using Chrome on my mobile device and I noticed that I cannot select the relevant passages whenever I judge a query result as "partial" or "perfect". Do I need any specific gestures to select the passages or is this not intended?

**Comment by sebastian-hofstaetter** [comment, marked as answer]
Hi, thanks for flagging this, we could reproduce it and also see many of your colleagues mentioning it in the feedback form. It is a bit of a puzzling error (as it worked before, and we don't know what's causing it). We'll look into it, but we can't promise that we will be able to fix it. For now you can just continue annotating and skip selecting answers on mobile as this is not the primary objective this year.
